firmware/common2015/drivers/kicker-board/KickerBoard.cpp

```diff
@@ -89,4 +89,61 @@ bool KickerBoard::flash(bool onlyIfDifferent, bool verbose) {
     return true;
 }
 
+bool KickerBoard::send_to_kicker(uint8_t cmd, uint8_t arg, uint8_t& ret_val,
+                                 bool verbose) {
+    chipSelect();
+    // Returns state (charging, not charging), but we don't care about that
+    uint8_t charge_resp = _spi->write(cmd);
+    // Should return the command we just sent
+    uint8_t command_resp = _spi->write(arg);
+    // Should return final response to full cmd, arg pair
+    ret_val = _spi->write(BLANK);
+    chipDeselect();
+
+    bool command_acked = command_resp == cmd;
+    if (verbose) {
+        if (!command_acked) {
+            printf("Kicker failed to ack command! -- ");
+        }
+
+        printf("Kicker: CHG:%02X, CMD:%02X, RET:%02X\r\n", charge_resp,
```

Comment:
Good point. I'm not completely sure how it will work within the hw-tests since I'm not too familiar with them. Both seem to be used interchangeably within the [radio-receiver-test](https://github.com/RoboJackets/robocup-firmware/blob/master/firmware/robot2015/hw-test/radio-receiver-test.cpp#L80). The only requirements for `LOG()` to work is for `isLogging` and `rjLogLevel` to be set. 

[`LOG()`](https://github.com/RoboJackets/robocup-firmware/blob/master/firmware/common2015/utils/logger/logger.cpp#L39) basically just surrounds the `printf()` with a mutex and adds a few "headers" such as time so it /should/ be the same as a normal `printf()`

If you want, the whole `if (verbose)` thing could be refactored to just `LOG(INF2, ...)` kinda like how it's done in the [mcp20317 class](https://github.com/RoboJackets/robocup-firmware/blob/b93c15c21918f78fd053ec0f146adade4f2cd878/firmware/common2015/drivers/mcp23017/mcp23017.cpp#L59). It looks to be the same type of data being printed out.